libs/api/src/lib/graphql/setting.ts

```diff
@@ -46,7 +46,15 @@ export const GraphQLSettingName = new GraphQLEnumType({
 export const GraphQLAllowedSettingVals = new GraphQLObjectType({
   name: 'AllowedSettingVals',
   fields: {
-    stringChoice: {type: GraphQLList(GraphQLString)},
+    stringChoice: {type: new GraphQLList(GraphQLString)},
+    boolChoice: {type: GraphQLBoolean}
+  }
+})
+
+export const GraphQLInputAllowedSettingVals = new GraphQLInputObjectType({
```

Comment:
Needs an InputObjectType now for inputs. Not a breaking change as only for admin API